apps/example/priv/repo/migrations/20161123003006_create_thesis_files_table.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Example.Repo.Migrations.CreateThesisFilesTable do
+  @moduledoc false
+  use Ecto.Migration
+
+  def change do
+    create table(:thesis_files) do
+      add :slug, :string
```

Comment:
@jamonholmgren doesn't path imply a way to find something in the directory structure? we aren't saving paths here (take a look at the router); we're saving a way to find the correct image in the table. Thoughts?